src/smtp.rs

```diff
@@ -364,7 +364,7 @@ pub(crate) async fn smtp_send(
     msg_id: MsgId,
 ) -> SendResult {
     if std::env::var(crate::DCC_MIME_DEBUG).is_ok() {
-        info!(context, "smtp-sending out mime message:");
+        info!(context, "SMTP-sending out mime message:");
         println!("{message}");
```

Comment:
Afaiu this prints the message to stdout, but the log above goes to stderr. Why do we need to print the message at all? If this is for debugging, DCC_MIME_DEBUG should be checked and `info!()` used as well